pubspec.yaml

```diff
@@ -39,7 +39,7 @@ dependencies:
   collection: ^1.17.2
   convert: ^3.1.1
   crypto: ^3.0.3
-  device_info_plus: ^10.0.1
+  device_info_plus: ^11.2.0
```

Comment:
Looks like the one "breaking" change in this changelog is:
> `BREAKING FIX(device_info_plus): fixed webasm compliance (#3254). (e35e2123)`

so I'll assume that's web-only and has no effect on us.